app/mailers/report_mailer.rb

```diff
@@ -8,7 +8,7 @@ def self.new_report(type, id)
   def new_report(type, id, role)
     resource = {
       url:  report_index_url,
-      type: I18n.t("notifier.report_email.type." + type),
+      type: I18n.t("notifier.report_email.type." + type.downcase),
```

Comment:
Wait, actually update this to `"notifier.report_email.type.#{type.downcase}"`, then squash.